content/source/docs/enterprise/api/run.html.md

```diff
@@ -388,6 +470,58 @@ curl \
   https://app.terraform.io/api/v2/runs/run-DQGdmrWMX8z9yWQB/actions/cancel
 ```
 
+## Forcefully cancel a run
+
+`POST /runs/:run_id/actions/force-cancel`
+
+Parameter | Description
+----------|------------
+`run_id`  | The run ID to cancel
+
+The `force-cancel` action is like [cancel](#cancel-a-run), but ends the run immediately. Once invoked, the run is placed into a `canceled` state, and the running Terraform process is terminated. The workspace is immediately unlocked, allowing further runs to be queued. The `force-cancel` operation requires workspace admin privileges.
+
+This endpoint enforces a prerequisite that a [non-forceful cancel](#cancel-a-run) is performed first, and a cool-off period has elapsed. To determine if this criteria is met, it is useful to check the `data.attributes.is-force-cancelable` value of the [run details endpoint](#get-run-details). The time remaining in the cool-off period can also be found using the [run details endpoint](#get-run-details), in the key `data.attributes.force_cancel_delay_seconds`. Note that this key is only present in the payload after the initial cancel has been initiated.
+
+This endpoint represents an action as opposed to a resource. As such, it does not return any object in the response body.
+
+-> **Note:** This endpoint cannot be accessed with [organization tokens](../users-teams-organizations/service-accounts.html#organization-service-accounts). You must access it with a [user token](../users-teams-organizations/users.html#api-tokens) or [team token](../users-teams-organizations/service-accounts.html#team-service-accounts).
+
+~> **Warning:** This endpoint has potentially dangerous side-effects, including loss of any in-flight state in the running Terraform process. Use this operation with extreme caution.
+
+Status  | Response                  | Reason(s)
+--------|---------------------------|----------
+[202][] | none                      | Successfully queued a cancel request.
+[409][] | [JSON API error object][] | Run was not planning or applying, has not been canceled non-forcefully, or the cool-off period has not yet passed.
```

Comment:
Should we have 404 here as well for permissions failure? (Is that what would happen?)

Reply:
It is - I noticed it was not documented in the cancel endpoint, so figured that permissions are probably talked about as a concept some place, and that 404 would have been mentioned there. 404 could be permissions but it could also be that the run was not found. Grepping around it looks like that's not the case and other endpoints do mention 404 so I'll just add it.